### PR TITLE
BZ-1684476: Added commands to delete all resources

### PR DESCRIPTION
**Pull request by bergerhoffer**
@openshift/team-documentation For peer review.

Preview: http://file.rdu.redhat.com/~ahoffer/2019/BZ-1684476/applications/operator_sdk/osdk-getting-started.html#building-memcached-operator-using-osdk_osdk-getting-started

### CODE REVIEW

**Comment by mburke5678** [comment]
@bergerhoffer LGTM

File: modules/building-memcached-operator-using-osdk.adoc [hunk 1]
```diff
@@ -325,6 +325,10 @@ example-memcached    4         4         4            4           5m
 . *Clean up the resources:*
 +
 ----
+$ kubectl delete -f deploy/crds/cache_v1alpha1_memcached_crd.yaml
```
Comment (jianzhangbjz) [inline modules/building-memcached-operator-using-osdk.adoc]: @bergerhoffer Sorry, my mistake. We should delete the `deploy/crds/cache_v1alpha1_memcached_crd.yaml` after deleting the `deploy/crds/cache_v1alpha1_memcached_cr.yaml`.

**Review by jianzhangbjz — CHANGES_REQUESTED** [review]

File: modules/building-memcached-operator-using-osdk.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/building-memcached-operator-using-osdk.adoc]: No problem, I've moved deleting `deploy/crds/cache_v1alpha1_memcached_crd.yaml` to below deleting `deploy/crds/cache_v1alpha1_memcached_cr.yaml`. Let me know if this looks okay now:

Preview: http://file.rdu.redhat.com/~ahoffer/2019/BZ-1684476/applications/operator_sdk/osdk-getting-started.html#building-memcached-operator-using-osdk_osdk-getting-started

**Comment by jianzhangbjz** [comment]
LGTM, thanks!

**Pull request merged**